File: test/plugin/spring/fixtures/general/src/main/java/inga/server/GetMappingDefinition.java
package inga.server;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RequestMapping("/mapping")
@RestController
public class GetMappingDefinition {
    @GetMapping
    public void getWithNoValue() {
    }

    @GetMapping("/{v}")
    public void getWithSingleMember() {
    }
}

File: test/plugin/spring/fixtures/kotlin/p1/server/spring/src/main/p1/GetMappingDefinition.kt
package p1

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RequestMapping("/mapping")
@RestController
class GetMappingDefinition {
    @GetMapping
    fun getWithNoValue() {
    }

    @GetMapping("/{v}")
    fun getWithSingleMember() {
    }
}
